chrome-os-best-practices/lib/main.dart

```diff
@@ -184,41 +183,44 @@ class ChatScreenState extends State<ChatScreen> with TickerProviderStateMixin {
         child: Row(
           children: <Widget>[
```

Comment:
nit (and not your code): `<Widget>` is inferred here.

I wish there were a linter for "avoid manually declaring a type that's already inferred," but I don't see anything that matches in the docs.

Reply:
Done, and me too.

/cc @bwilkerson =)

Reply:
Request recorded: https://github.com/dart-lang/linter/issues/1634. Feel free to add to the issue.

Reply:
This makes me very happy.